src/shared/components/login/login.js

```diff
@@ -56,15 +56,20 @@ class Login extends Component {
   }
 
   render() {
-    const { error } = this.state;
+    let { error } = this.state;
```

Comment:
I can appreciate the idea of overriding state with a copy variable, but with this implementation it's a bit confusing - I thought at first you were trying to directly mutate state.

Instead, let's name `error` as `errorMessage` for the sake of clarity.